### PR TITLE
SDK update to v2024.2.1 + new create_sub_org example

### PR DESCRIPTION
**Pull request by r-n-o**
This branch does a few things:
* update our swagger spec to reflect what's currently in production
* update the type of swagger spec to be the external spec (without the version prefixes, much cleaner!)
* add a quick example to showcase sub-org creation and Tron address capabilities.
* bump gem to v0.0.2 as a result

### CODE REVIEW

File: examples/create_sub_org/create_sub_org.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              addressFormat: 'ADDRESS_FORMAT_TRON'
+            }
+          ]
+        }
+      }
+    }
+  )
+
+  created_sub_organization = create_sub_organization_response&.activity&.dig(:result, :createSubOrganizationResultV4)
+  # TODO
```
Comment (andrewkmin) [inline examples/create_sub_org/create_sub_org.rb]: what's the TODO here? 😄

File: examples/create_sub_org/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+To run this example:
+* `cp .env.example .env`
+* Follow the instructions in the `.env` file to fill in values for `TURNKEY_ORGANIZATION_ID`, `TURNKEY_API_PUBLIC_KEY`, `TURNKEY_API_PRIVATE_KEY`, and `SUB_ORGANIZATION_ROOT_PUBLIC_KEY`
+* Install the dependencies: `bundle install`
+* Then run the `create_sub_org.rb` script: `bundle exec ruby create_sub_org.rb`
+
+Here's an example run:
+
+```
+bundle exec ruby create_sub_org.rb
```
Comment (andrewkmin) [inline examples/create_sub_org/README.md:14]: ahhh the good ol bundle exec :')

**Review by andrewkmin — APPROVED** [review]
👏💎

File: examples/create_sub_org/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Who am I?
+
+This examples shows how to use [`turnkey_client`](https://rubygems.org/gems/turnkey_client) to create a new sub-organization and a wallet. We generate a new Tron account (default account)
+
+To run this example:
+* `cp .env.example .env`
+* Follow the instructions in the `.env` file to fill in values for `TURNKEY_ORGANIZATION_ID`, `TURNKEY_API_PUBLIC_KEY`, `TURNKEY_API_PRIVATE_KEY`, and `SUB_ORGANIZATION_ROOT_PUBLIC_KEY`
```
Comment (andrewkmin) [inline examples/create_sub_org/README.md]: should we mention anything about shadowing here? and/or make it explicitly clear that `TURNKEY_API_PUBLIC_KEY` and `TURNKEY_API_PRIVATE_KEY` should correspond with the parent org?

**Review by AlwaysBCoding — APPROVED** [review]
don't see anything that stands out with the create_sub_org.rb example, looks good

File: examples/create_sub_org/create_sub_org.rb (hunk 1 above)
Comment (r-n-o) [reply examples/create_sub_org/create_sub_org.rb]: whoops. Nothing! That was just a marker for me, but forgot to remove it 🙈

File: examples/create_sub_org/README.md (hunk 3 above)
Comment (r-n-o) [reply examples/create_sub_org/README.md]: Yeah I'll add a blurb about this 👍

Comment (r-n-o) [reply examples/create_sub_org/README.md]: done! I also added a runtime check to ensure these public keys are different.

**Pull request merged**